lib/rmg/CRmgTemplate.h

```diff
@@ -41,17 +41,24 @@ namespace EWaterContent
 	};
 }
 
+namespace EZoneMonsterStrength
+{
+	enum EZoneMonsterStrength
+	{
+		WEAK,
+		NORMAL,
+		STRONG
+	};
+}
+
 namespace EMonsterStrength
 {
 	enum EMonsterStrength
```

Comment:
To be consistent, I'd rename this to `EGlobalMonsterStrength`.

Reply:
That was my first thought too, but it's also used on screens in game and in editor and in those contexts EZoneMonsterStrength doesn't appear, so I thought that maybe it's better to leave the old name. I mean, random map selection screen doesn't know other monster strengths than global, so it's fine with the old name. But I can easily rename all uses of EMonsterStrength to EGlobalMonsterStrength if you think that would look better.

Reply:
Ok, the name EMonsterStrength does feel wrong when there's a EZoneMonsterStrength nearby. I'll rename in a moment as you suggested.

Reply:
Done.